MaxMind.GeoIP2/Responses/EnterpriseResponse.cs

```diff
@@ -9,13 +9,10 @@
 namespace MaxMind.GeoIP2.Responses
 {
     /// <summary>
-    ///     This class provides a model for the data returned by the GeoIP2 Precision:
-    ///     Insights end point.
-    ///     The only difference between the City and Insights response classes is
-    ///     which fields in each record may be populated.
+    ///     This class provides a model for the data returned by the GeoIP2 Enterprise
+    ///     database.
     ///     <a href="https://dev.maxmind.com/geoip/docs/web-services?lang=en">
-    ///         GeoIP2 Web
-    ///         Services
+    ///         GeoIP2 Web Services
```

Comment:
Why do we link to the web services?

Reply:
Not sure. All of the APIs do. I think it is so that they can refer to that documentation. I do think most of them say something like "See..." though.

Reply:
Oh, I see. You are commenting on the Enterprise class in particular. :shrug: I think I will just remove them from all the response classes though. It seems like noise and is confusing as we aren't linking to the database docs.